src/main/java/com/kenny/openimgur/widget/GalleryWidgetProvider.java

```diff
@@ -0,0 +1,134 @@
+package com.kenny.openimgur.widget;
+
+import android.app.PendingIntent;
+import android.appwidget.AppWidgetManager;
+import android.appwidget.AppWidgetProvider;
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.text.TextUtils;
+import android.widget.RemoteViews;
+
+import com.kenny.openimgur.R;
+
+/**
+ * Widget provider which tells the launcher to update the widgets
+ * Created by John on 4/3/2015.
+ */
+public class GalleryWidgetProvider extends AppWidgetProvider {
+
+    public static final String ACTION_NEXT = "ACTION_NEXT";
+    public static final String ACTION_PREVIOUS = "ACTION_PREVIOUS";
+    public static final String ACTION_VIEW = "ACTION_VIEW";
+
+    public static final String EXTRA_ITEM = "EXTRA_ITEM";
+
+    @Override
+    public void onDeleted(Context context, int[] appWidgetIds) {
```

Comment:
Do these override methods need to be include even though you don't do anything asides from the super call?